ttnn/cpp/ttnn/operations/eltwise/binary/device/binary_composite_op.cpp

```diff
@@ -103,7 +103,7 @@ Tensor _isclose(
     Tensor mul_result = ttnn::multiply(ttnn::abs(value2, output_mem_config), rtol, std::nullopt, output_mem_config);
     is_close_rhs = ttnn::add(mul_result, atol, std::nullopt, output_mem_config);
     mul_result.deallocate();
-    Tensor result = ttnn::where(ttnn::le(is_close_lhs, is_close_rhs, std::nullopt, output_mem_config), 1.0, 0.0);
+    Tensor result = ttnn::where(ttnn::le(is_close_lhs, is_close_rhs, std::nullopt, output_mem_config), 1.0f, 0.0f);
```

Comment:
Changes like this were caused by changing from overloads to using `std::variant<float, Tensor>` parameter. The rule in C++ is one implicit conversion. `double` -> `float` was fine (because we have certain warnings turned off) but `double` -> `std::variant<float, Tensor>` requires two implicit conversions so it's not allowed.